src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File fileToRead = new File(fromFileName);
+        File fileToWrite = new File(toFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        int buyScore = 0;
+        int supplyScore = 0;
+        String infoTostring = readToFile(fileToRead, stringBuilder);
+        buyScore = calculateCount(infoTostring, buyScore, BUY);
+        supplyScore = calculateCount(infoTostring, supplyScore, SUPPLY);
+        int resultScore = supplyScore - buyScore;
+        writeToFile(fileToWrite, buyScore,supplyScore,resultScore);
```

Comment:
Make getStatistic like an algorithm
```suggestion
        String dataFromFile = readFile(fromFileName);
        String report = createReport(dataFromFile);
        writeToFile(report, toFileName);
```